envs/monkey_zoo/docs/zoo_setup.md

```diff
@@ -0,0 +1,119 @@
+# Setting up a Monkey Zoo using Terraform
+
+
+
+## Getting started:
+
+Requirements:
+1.  Have terraform installed (for installation instructions check this
+    [link](https://learn.hashicorp.com/tutorials/terraform/install-cli)).
+2.  Have a Google Cloud Platform account (upgraded if you want to test
+    whole network at once).
```

Comment:
Gcloud cli is required as well?

Reply:
Do we need to specify that `envs/monkey_zoo` folder needs to be downloaded and all actions are done in reference to it? These files can be viewed online in git